services/src/main/scala/cromwell/services/healthmonitor/impl/workbench/WorkbenchHealthMonitorServiceActor.scala

```diff
@@ -57,19 +53,6 @@ abstract class WorkbenchHealthMonitorServiceActor(val serviceConfig: Config,
         throw new IllegalArgumentException("Unable to configure WorkbenchHealthMonitor: " + e.toList.mkString(", "))
     }
 
-  /**
-    * Demonstrates connectivity to GCS by stat-ing a bucket
-    */
-  private def checkGcs(): Future[SubsystemStatus] = {
-    // For any expected production usage of this check, the GCS bucket should be public read */
```

Comment:
I think this comment was incorrect, the HC buckets in Terra projects are not public.

```
> gcloud storage buckets describe gs://cromwell-ping-me-dev
ERROR: (gcloud.storage.buckets.describe) [oednichols@gmail.com] does not have permission to access b instance [cromwell-ping-me-dev] (or it may not exist): oednichols@gmail.com does not have storage.buckets.get access to the Google Cloud Storage bucket.
```